.github/workflows/labeler.yml

```diff
@@ -5,6 +5,9 @@ on:
 jobs:
   labeler:
     runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
```

Comment:
Why was this changed?

Reply:
was changed as an attempt to get the `Pull Request Labeler` test to pass but it seems like an issue on the repo level. Reverted for now